README.md

```diff
@@ -38,23 +34,14 @@ Various files are used to configure and deploy the Lambda.
 
 `.env` is used *locally* for two purposes:
```

Comment:
I'd remove "*locally*" since it's now in source control?

Reply:
Fixed